code/cmdline/cmdline.cpp

```diff
@@ -886,6 +892,11 @@ bool has_cmdline_only_flag(int argc, char *argv[])
 void os_init_cmdline(int argc, char *argv[])
 {
 	FILE *fp;
+
+#if defined(APPLE_APP)
+	char *path_name = SDL_GetBasePath();
```

Comment:
The SDL doc says that the pointer needs to be freed with `SDL_free`. The current version would cause a memory leak.